haskell/src/Ligo/BaseDAO/RegistryDAO/Types.hs

```diff
@@ -88,4 +88,4 @@ instance HasAnnotation RegistryExtra where
 type instance VariantToParam 'Registry = RegistryCustomEpParam
 type instance VariantToExtra 'Registry = RegistryExtra
 
-type RegistryFullStorage = FullStorageSkeleton (VariantToExtra 'Registry)
+type RegistryFullStorage = StorageSkeleton (VariantToExtra 'Registry)
```

Comment:
Should be renamed as well as, `TreasuryFullStorage`.

Reply:
Changed